gsonpath-sample/src/main/java/gsonpath/generated/PersonModelGenerated.java

```diff
@@ -5,8 +5,9 @@
 
 import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)